### DISCUSSION TITLE
Debugging the app

### DISCUSSION THREAD

**Discussion by flaviu22** [cnr-isti-vclab/meshlab#878] [Q&A] [answered]
I hesistate to write here such a thing: how can I debug the meshlab app, not **use_cpu_opengl** ?

I have successfully compiled the entire app:

![image](https://user-images.githubusercontent.com/37621568/102972796-497c4a00-4504-11eb-928e-6bda7fe08528.png)

but when I try to start debug, is always start **use_cpu_opengl**, not meshlab. And I didn't saw an option to tell what would be the starting project. Sorry for this trivial question, I am using Qt for very few time.

**Comment by alemuntoni** [comment, marked as answer]
In the left bar (at the bottom, where it is written "Debug"), you can choose to build in debug or release mode, and you can choose between all the target binaries of the project (under the "Run" column).

**Comment by flaviu22** [comment]
Thank you ! Is obvious now ! 👍
